admin/tabs/activity/tracking/detail/ActivityDetailModel.js

```diff
@@ -56,8 +56,10 @@ export class ActivityDetailModel extends HoistModel {
                     align: 'center',
                     width: 50,
                     renderer: (v, {record}) => {
-                        const {impersonating} = record.data;
-                        return impersonating ?
+                        const {impersonating} = record.data,
+                            isImpersonating = impersonating !== null;
+
+                        return isImpersonating ?
```

Comment:
would be just as clear to just check the truthiness of `impersonating` as before -- that seems like standard javascript style here....